inc/Recipe.php

```diff
@@ -13,11 +13,11 @@ class Recipe {
   public $components = [];
 
   public function get_components() {
-    if (empty($this->componenets)) {
+    if (empty($this->components)) {
       // idk
     }
 
-    return $this->componenets;
+    return $this->components;
```

Comment:
You have these in PR #3 , so you shouldn't include them here, or you should remove them from the other PR.